gkeepapi/__init__.py

```diff
@@ -410,16 +410,114 @@ def __init__(self, auth=None):
             },
         }
 
-    def create(self):
+    def create(self,note_id, note_server_id, date_str = "2020/1/21", time_str = "08:22:33"):
```

Comment:
I'd prefer to have a `datetime.datetime` instead.

Reply:
Also, could you fix up the spacing too?
```suggestion
    def create(self, note_id, note_server_id, date_str = "2020/1/21", time_str = "08:22:33"):
```

Reply:
Have changed it to a datetime.datetime object instead. I also fixed the spacing